assets/css/style.css

```diff
@@ -0,0 +1,224 @@
+    opacity: 0;
+}
+
+#hamburger-menu.active .bar-bottom {
+    transform: translate(-50%, -50%) rotate(-225deg);
+    top: 50%;
+}
+
+/* off Screen */
+.off-screen {
+    position: fixed;
+    bottom: -100%;
+    height: 100%;
+    width: 100%;
+    left: 0px;
+    z-index: 99;
+    background-color: var(--off-screen-background-color);
+    transition: bottom .3s;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+}
+
+.off-screen.active {
+    bottom: 0;
+}
+
+.menu {
+    list-style-type: none;
+    text-align: center;
+    margin-top: 150px;
+}
+
+.menu .menu-item a {
+    text-decoration: none;
+    color: var(--text-primary-color);
+    font-family: var(--web-font);
+    letter-spacing: -1px;
+    font-size: 50px;  
+}
+
+.bottom {
+    display: flex;
+    margin-top: auto;
+}
+
+.bottom-icon {
+    display: flex;
+    padding-left: 650px;
+}
+
+.bottom-text .a-tag {
+    color: var(--text-primary-color);
+    text-decoration: none;
+}
+
+.bottom-icon .a-tag {
+    margin-left: 20px;
+    color: var(--text-primary-color);
+}
+
+/* banner Style */
+.banner {
+    background-color: var(--web-background);
+    height: 100vh;
+}
+
+.banner-image {
+    background-image: url('../img/vaynerbreakthrough2022.jpeg');
+    height: 100vh;
+    background-position: right;
+    background-repeat: no-repeat;
+}
+
+.ad-week-img {
+    margin-top: 25px;
+    margin-bottom: 20px;
+    max-width: 250px;
+    max-height: 100px;
+}
+
+.agency-text mark {
+    /* background-color: var(--off-screen-background-color) !important; */
+    background-image: linear-gradient(to right,#27ccc0 0,#27ccc0 100%);
+    color: var(--text-primary-color);
+    font-family: var(--web-font);
+    font-size: x-large;
+    letter-spacing: 1px;
+    padding: 0;
+    text-transform: uppercase;
+}
+
+.story-text {
+    text-transform: uppercase;
+    font-family: var(--permanent-marker-font);
+    color: var(--text-primary-color);
+    font-size: 25px;
+    letter-spacing: 1px;
+    margin-top: 30px;
+    margin-bottom: 0;
+}
+
+.auther-text {
+    color: var(--text-primary-color);
+    font-size: 20px;
+    margin-left: 8rem;
+    margin-top: 0px;
+}
+
+.about-text {
+    text-transform: uppercase;
+    color: var(--text-primary-color);
+    font-family: var(--web-font);
+    letter-spacing: 1px;
+    font-size: 20px;
+    margin-bottom: 30px;
+}
+
+.read-btn {
+    text-decoration: none;
+    color: var(--off-screen-background-color);
+    text-transform: uppercase;
+    font-size: small;
+    padding: 8px 25px 8px 35px;
+    font-weight: bold;
+    border: 2px solid var(--off-screen-background-color);
+    transition: all ease .5s;
+}
+.read-btn i {
+    visibility: hidden;
+}
+
+.read-btn:hover {
+    color: var(--text-primary-color);
+    background-color: var(--off-screen-background-color);
+}
+
+.read-btn:hover i {
+    visibility: visible;
+    margin-left: 10px;
+}
+
+@media screen and (max-width: 999px) {
```

Comment:
Your media queries should always follow the bootstrap standard. It should either be 1024px, 992px, 1200px or 768px. Do not create haphazard numbers for media queries